dll/cpl/hdwwiz/lang/pl-PL.rc

```diff
@@ -78,7 +79,7 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "Kreator dodawania sprzętu"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Jeśli nie widzisz odpowiedniej kategorii urządzenia kliknij przycisk Pokaż wszystkie urządzenia.", IDC_STATUSTEXT, 21, 9, 275, 18
+    LTEXT "Jeśli nie widzisz odpowiedniego sprzętu, kliknij przycisk Pokaż wszystkie urządzenia.", IDC_STATUSTEXT, 21, 9, 275, 18
```

Comment:
From Win 10:
```suggestion
    LTEXT "Jeśli nie widzisz odpowiedniej kategorii sprzętu, kliknij przycisk Pokaż wszystkie urządzenia.", IDC_STATUSTEXT, 21, 9, 275, 18
```

Reply:
ReactOS does not aim to have a 100% copy-paste of the translations.